guides/release/getting-started/quick-start.md

```diff
@@ -222,6 +222,8 @@ other parts of the app we are building.
 We've also renamed `scientist` to the more-generic `person`,
 decreasing the coupling of our component to where it's used.
 
+Also, note that ember strictly follows `title case naming` style in templates, so defining `<PeopleList>` as `<peoplelist>` or `<peopleList>` won't work.
```

Comment:
In the section [Introducing Components](https://guides.emberjs.com/release/components/introducing-components/), I found a "Zoey says" that advises readers to check casing. I liked that it drew a reader's attention, but found its message to be a bit unclear:

> <img width="640" alt="introducing_components" src="https://user-images.githubusercontent.com/16869656/90337442-12bb8380-dfa8-11ea-922c-7f85bce8ed55.png">

Can we change your line to the following:

```suggestion
Our component is called `<PeopleList>`, based on its name on the file system. Please check that the letters P and L are capitalized.

<div class="cta">
  <div class="cta-note">
    <div class="cta-note-body">
      <div class="cta-note-heading">Zoey says...</div>
      <div class="cta-note-message">
        A component's name is derived from its file name.
        We capitalize the first letter and every letter after <code>-</code>, then remove the hyphens.
        This is known as pascal case.
      </div>
    </div>
    <img src="/images/mascots/zoey.png" role="presentation" alt="">
  </div>
</div>
```

I think these lines can address both your concern as well as mine.

If so, afterwards, will you update the "Zoey says" in the Introducing Component section to use the `<div>` code above?

Reply:
Your sentence is well worded and it does clarifies my concerns. I would be happy to help updating zoey says. Thankyou for the suggesting.

Reply:
Your sentence is well worded and it does clarifies my concerns. I would be happy to help updating zoey says. Thankyou for the suggestion